src/vochain/vochain.proto

```diff
@@ -58,21 +87,14 @@ message CAbundle {
         bytes address = 2;
 }
 
-message ProofEthereumStorage {
-        bytes key = 1;
-        bytes value = 2;
-        repeated bytes siblings = 3;
+message ProofArboPoseidon {
```

Comment:
I think we could join both Arbo Poseidon proofs and add an extra field for type.